api/v1beta1/disruption_types.go

```diff
@@ -747,7 +751,7 @@ func (s DisruptionSpec) validateGlobalDisruptionScope(requireSelectors bool) (re
 		}
 	}
 
-	if s.GRPC != nil && s.Level != chaostypes.DisruptionLevelPod {
+	if s.GRPC != nil && s.Level == chaostypes.DisruptionLevelNode {
```

Comment:
Turns out that the "defaults" aren't appropriately set when we directly call Spec.Validate(), so grpc didn't realize an empty string meant pods.